Sources/StrongContractResponse/All/StringProtocol.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  File.swift
+//  
+//
+//  Created by lsd on 07/05/24.
+//
+
+import Foundation
+//
+//  File.swift
+//
+//
+//  Created by Leonardo Dabus on 04/07/21.
+//
+
+import Foundation
+import protocol Foundation.DataProtocol
+import protocol Foundation.ContiguousBytes
+import struct Foundation.Data
+
+#if canImport(Crypto)
+import Crypto
+#endif
+
+#if canImport(CryptoKit)
+import CryptoKit
+#endif
+
+extension StringProtocol {
+
+    var data: Data { .init(utf8) }
+    var hexaData: Data { .init(hexa) }
```

Comment:
Seems like we shouldn't have to manually calculate this.  Is there really no system function that can do this?